index.html

```diff
@@ -62,9 +62,13 @@ <h1>Launch Guide</h1>
                         Click on the link below and invite the team
                         members who directly report to you.
                     </div>
-                    <button class="btn btn-warning mt-3">
-                        Invite Your Team
-                    </button>
+                    <a href="invite_your_team.html">
+                        <button class="btn btn-warning mt-3">
+                            <strong>
+                                Invite Your Team
+                            </strong>
```

Comment:
Do we need `<strong>` tag for button text when we defined it in styles?
https://github.com/codemakeracademy/weekly-team-report-html/blob/94b83de4a56cfb673dd9f7173e5d73da8e687da9/src/styles/_common.scss#L11-L13